robust_pdf_plot.ipynb

```diff
@@ -0,0 +1,311 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #97.        n = norm.pdf(x_axis, loc=loc, scale=scale)
</pre><p>I suggest to use meaning ful variable name (PEP8 naming convention) probably you can use.</p><p>n -&gt; norm_pdf</p><p>l-&gt; laplace_pdf </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Updated variable names as suggested above</p>